src/chttpd/src/chttpd_prefer_header.erl

```diff
@@ -22,6 +22,11 @@
 -include_lib("couch/include/couch_db.hrl").
 
 
+-define(DEFAULT_PREFER_MINIMA,
```

Comment:
Typo. It should be `DEFAULT_PREFER_MINIMAL`.

Reply:
Changed.